src/containers/Chat.tsx

```diff
@@ -506,6 +507,13 @@ export class ChatClient extends React.Component<Props, State> {
         return
       }
 
+      const userId = parsedMessage.user.id;
+      const indexOfCompressedUser = this.props.compressedIds.indexOf(userId);
+
+      if (indexOfCompressedUser > -1) {
+        serializedMessage.compressed = true
+      }
```

Comment:
The serialized message should not be modified here but in `libs/Message` instead by padding new parsing options if needed.